expo/gbe/models/biddable.py

```diff
@@ -12,34 +12,36 @@
 from django.db.models import Q
 
 
-visible_bid_query = (Q(biddable_ptr__conference__status='upcoming') |
-                     Q(biddable_ptr__conference__status='ongoing'))
+visible_bid_query = (Q(biddable_ptr__b_conference__status='upcoming') |
+                     Q(biddable_ptr__b_conference__status='ongoing'))
 
 
 class Biddable(Model):
     '''
     Abstract base class for items which can be Bid
     Essentially, specifies that we want something with a title
     '''
-    title = CharField(max_length=128)
-    description = TextField(blank=True)
+    b_title = CharField(max_length=128)
+    b_description = TextField(blank=True)
     submitted = BooleanField(default=False)
     accepted = IntegerField(choices=acceptance_states,
                             default=0,
                             blank=False)
     created_at = DateTimeField(auto_now_add=True)
     updated_at = DateTimeField(auto_now=True)
-    conference = ForeignKey(
+    b_conference = ForeignKey(
```

Comment:
These are the root of most of the changes  - the conflict between Biddable and Event when inherited by class meant that we had to change the fields to deconflict the models.